helpers/query.go

```diff
@@ -75,5 +77,13 @@ func QueryTxs(chain *relayer.Chain, eventsStr string, offset uint64, limit uint6
 		return nil, err
 	}
 
+	if offset > math.MaxInt64 {
```

Comment:
I think this should be MaxInt32? right? I believe int is 32 bytes though apparently not an alias for int32

Reply:
Okay, but I think int is atleast 32 bytes. It's just depends on platform and can be 64 bits I guess. Is it right?

Reply:
that sounds correct. Let's leave as is then